hexrd/ui/image_file_manager.py

```diff
@@ -75,11 +75,19 @@ def load_aps_imageseries(self, detectors, directory_names):
                 return
 
     def open_file(self, f):
-        ext = os.path.splitext(f)[1]
-        if self.is_hdf5(ext):
-            ims = imageseries.open(f, 'hdf5',
-                path=self.path[0],
-                dataname=self.path[1])
+        ext = os.path.splitext(f)[1] if isinstance(f, str) else None
```

Comment:
What other type can `f` be?

Reply:
`f` may not be a file at all but instead a `numpy` array that needs to be converted to an image series. [Here](https://github.com/HEXRD/hexrdgui/blob/ebbc5e368788e2ba4b9a1480a4a1669044890624/hexrd/ui/import_data_panel.py#L173) the masked image is passed as an array an an image series is created from the processed raw image (and then cropped). Maybe this would be better factored out into its own function?

Reply:
OK, lets just add a comment indicating the the two possible types.